ui/src/app/cohort-search/cohort-page/cohort-page.component.tsx

```diff
@@ -14,18 +14,20 @@ import {LOCAL_STORAGE_KEY_COHORT_CONTEXT} from 'app/pages/data/criteria-search';
 import {cohortsApi} from 'app/services/swagger-fetch-clients';
 import colors from 'app/styles/colors';
 import {
+  hasNewValidProps,
   reactStyles,
   withCurrentCohortSearchContext,
   withCurrentWorkspace
 } from 'app/utils';
 import {
   currentCohortSearchContextStore,
-  currentCohortStore,
-  queryParamsStore
+  currentCohortStore
 } from 'app/utils/navigation';
 import {WorkspaceData} from 'app/utils/workspace-data';
 import {Cohort, SearchRequest} from 'generated/fetch';
-import {Prompt} from 'react-router';
+import * as querystring from 'querystring';
```

Comment:
We already depend on this package, but it looks to be deprecated https://www.npmjs.com/package/querystring . I also prefer `URLSearchParams` if possible here.